packages/databricks-vscode/src/configuration/ui/ConfigurationDataProvider.ts

```diff
@@ -48,6 +50,9 @@ export class ConfigurationDataProvider
         );
 
         this.disposables.push(
+            this.bundleProjectManager.onDidChangeStatus(() => {
+                this._onDidChangeTreeData.fire();
+            }),
```

Comment:
I see why this event is needed now, but is there a way to maybe dispose this class from the BundleProjectManager and then recreate it when we have a project? I am still uncomfortable with exporting this event. 

Not the highest priority immediately, but definitely would be careful of taking anymore dependencies on the init class.

Reply:
Currently bundle project manager is not really an "init" class. For that we need to move a lot of initialisation and creation of classes into it, plus add disposal of them when we detect that the bundle config has disappeared. I would prefer to do it in one of the follow ups that focuses only on such restructuring though